redis.tf

```diff
@@ -33,3 +33,7 @@ resource "aws_elasticache_replication_group" "this" {
   num_cache_clusters         = 2
   apply_immediately          = true
 }
+
+locals {
+  auth_token = var.enforce_ssl ? random_password.auth_token.result : null

```

Comment:
Enforcing ssl is not the same as configuring redis with an auth token.

Reply:
According to the docs and the error I received, if I turn off transit_encryption_enabled, the auth_token parameter can't be set. Are you saying to perhaps rename the variable?

Reply:
So how do I disable ssl then?

Reply:
Is ssl causing a problem or the use of an auth token?

Reply:
If he tries to connect using the ssl enabled url `rediss://:-jv<7AP-fH&Bwd4EV8U!s9A<@master.taupe-armadillo-wocug.yax0kq.usw2.cache.amazonaws.com:6379/0` he is getting the following error:

Error -2 connecting to redis:6379. Name does not resolve.

When he tries a url without ssl `redis://:-jv<7AP-fH&Bwd4EV8U!s9A<@master.taupe-armadillo-wocug.yax0kq.usw2.cache.amazonaws.com:6379/0` he is getting the following error:

A Redis SSL connection URL should use the scheme rediss://

I found some docs where it mentioned having to configure a cert to go with the connection.
https://github.com/celery/celery/issues/5371

Instead of fighting this, I thought it would just be easier to add a quick option so that he can just connect over a non-TLS connection (redis://).